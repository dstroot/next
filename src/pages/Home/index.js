import React, { useEffect } from 'react';
import './styles.scss';

// Components
import Section from '../../components/Section';
import SectionBestLife from '../../components/SectionBestLife';
import SectionManageUncertainty from '../../components/SectionManageUncertainty';
import SectionFirstProduct from '../../components/SectionFirstProduct';
import SectionDesignWithYouInMind from '../../components/SectionDesignWithYouInMind';
import SectionAskUs from '../../components/SectionAskUs';

const Home = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `NEXT â€¢ Home`;
    // window.scrollTo(0, 0);
  });

  return (
    <>
      <Section />
      <SectionBestLife />
      <SectionManageUncertainty />
      <SectionFirstProduct />
      <SectionDesignWithYouInMind />
      <SectionAskUs />
    </>
  );
};

export default Home;
