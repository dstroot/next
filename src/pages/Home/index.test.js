import React from "react";
import { create } from "react-test-renderer";
import Home from "../Home";

describe("Home", () => {
  it("it should render", () => {
    let tree = create(<Home />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
