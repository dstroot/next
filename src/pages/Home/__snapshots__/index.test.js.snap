// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home it should render 1`] = `
Array [
  <div>
    <div
      className="background"
      style={
        Object {
          "backgroundImage": "url(city.svg)",
        }
      }
    >
      <div
        className="divText"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col text-center"
            >
              <h1
                className="nxt_heading-large"
              >
                What’s your next adventure?
              </h1>
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="col-md-8 offset-md-2 text-center"
            >
              <p
                className="mt-4 circular"
              >
                Insurance products to complement your life goals and passions. We help you go after whatever's next for you.
              </p>
              <a
                className="text-decoration-none mt-4"
                href="#bestlife"
              >
                <p
                  className="mt-4 mb-2 text-uppercase text-primary"
                >
                  Learn More
                </p>
                <img
                  alt="Arrow"
                  className="d-block-inline"
                  src="arrowdown.svg"
                  width={45}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <img
        alt="moped"
        className="hero-image"
        id="moped"
        src="moped.svg"
      />
    </div>
  </div>,
  <div
    id="bestlife"
  >
    <div
      className="py-6 bg-light"
    >
      <div
        className="container text-left"
      >
        <div
          className="container undefined"
        >
          <h1
            className="react-reveal  nxt_heading-xlarge undefined "
            style={
              Object {
                "opacity": undefined,
              }
            }
          />
          <div
            className="row"
          >
            <div
              className="react-reveal col-md-5 undefined"
              style={
                Object {
                  "opacity": undefined,
                }
              }
            >
              <img
                alt="Yoga Lady"
                className="img-fluid p-5"
                src="yogalady.svg"
              />
            </div>
            <div
              className="col-md-7 undefined"
            >
              <div
                className="mt-5"
              >
                <h1
                  className="react-reveal nxt_heading-xlarge"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                >
                  Keep living your best life
                </h1>
                <div
                  className="react-reveal mt-4 pr-5"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                >
                  Everyone deserves a chance to do what they love. Next by Pacific Life helps you reach financial goals while you go after your personal ones...
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div>
    <div
      className="py-6 undefined"
    >
      <div
        className="container text-left"
      >
        <div
          className="container undefined"
        >
          <h1
            className="react-reveal  nxt_heading-xlarge undefined "
            style={
              Object {
                "opacity": undefined,
              }
            }
          />
          <div
            className="row"
          >
            <div
              className="react-reveal col-md-5 order-2"
              style={
                Object {
                  "opacity": undefined,
                }
              }
            >
              <img
                alt="Canoe Couple"
                className="img-fluid p-5"
                src="canoecouple.svg"
              />
            </div>
            <div
              className="col-md-7 order-1"
            >
              <div
                className="mt-5"
              >
                <h1
                  className="react-reveal nxt_heading-xlarge"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                >
                  A way to manage uncertainty
                </h1>
                <div
                  className="react-reveal mt-4 pr-5"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                >
                  Plans change over time—your insurance products can too. Redefine your insurance products as life happens.
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="py-6 bg-light"
  >
    <div
      className="container text-center"
    >
      <h1
        className="react-reveal  nxt_heading-xlarge "
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        Introducing our first product
      </h1>
      <h1
        className="react-reveal nxt_heading-large mb-5"
        style={
          Object {
            "opacity": undefined,
          }
        }
      />
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="card mx-auto my-5 card border-dark mb-3 nxt_rounded-corner p-3 nxt_card-width"
        >
          <div
            className="card-body text-center"
          >
            <img
              alt="Evergreen"
              className="mb-2"
              src="evergreen.svg"
            />
            <h5
              className="card-title nxt_heading-small mb-4"
            >
              Next Deferred Income Annuity
            </h5>
            <p
              className="card-text circular mb-5"
            >
              Create a future income stream you can count on for the rest of your life by saving over time.
            </p>
            <a
              href="/nextdeferredincomeannuity"
              onClick={[Function]}
            >
              <button
                className="btn btn-primary rounded-pill nxt_body-small px-4 py-2"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div>
    <div
      className="py-6 undefined"
    >
      <div
        className="container text-left"
      >
        <div
          className="container text-center"
        >
          <h1
            className="react-reveal  nxt_heading-xlarge nxt_heading-large mb-5 "
            style={
              Object {
                "opacity": undefined,
              }
            }
          >
            Designed with you in mind
          </h1>
          <div
            className="row"
          >
            <div
              className="react-reveal col-md-5 undefined"
              style={
                Object {
                  "opacity": undefined,
                }
              }
            >
              <img
                alt="Woman with the Phone"
                className="img-fluid p-5"
                src="womanphone.svg"
              />
            </div>
            <div
              className="col-md-7 undefined"
            >
              <div
                className="mt-5"
              >
                <h1
                  className="react-reveal nxt_heading-xlarge"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                />
                <div
                  className="react-reveal mt-4 pr-5"
                  style={
                    Object {
                      "opacity": undefined,
                    }
                  }
                >
                  <div>
                    <h2
                      className="react-reveal nxt_heading-small mb-2"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                       Fearlessly Flexible
                    </h2>
                    <p
                      className="react-reveal mb-5"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                      Our products adapt to your life’s biggest moments. Change them when your needs do.
                    </p>
                    <h2
                      className="react-reveal nxt_heading-small mb-2"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                      Radically Simple
                    </h2>
                    <p
                      className="react-reveal mb-5"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                      Straight forward products and features. Know what you're getting from the beginning.
                    </p>
                    <h2
                      className="react-reveal nxt_heading-small mb-2"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                      Brazenly Transparent
                    </h2>
                    <p
                      className="react-reveal mb-5"
                      style={
                        Object {
                          "opacity": undefined,
                        }
                      }
                    >
                      No hidden fees or charges.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="py-6 bg-light"
  >
    <div
      className="container text-left"
    >
      <div
        className="container text-center"
      >
        <h1
          className="react-reveal  nxt_heading-xlarge nxt_heading-large "
          style={
            Object {
              "opacity": undefined,
            }
          }
        >
          Ask us anything
        </h1>
        <div
          className="row"
        >
          <div
            className="react-reveal col-md-5 order-2 mt-5 text-left"
            style={
              Object {
                "opacity": undefined,
              }
            }
          >
            <div>
              <h2>
                Next by Pacific Life
              </h2>
              <a
                href="mailto: hello@meetnext.com"
              >
                hello@meetnext.com
              </a>
              <p>
                (833) 646-NEXT (6398) 
              </p>
            </div>
          </div>
          <div
            className="col-md-7 order-1 text-left"
          >
            <div
              className="mt-5"
            >
              <h1
                className="react-reveal nxt_heading-xlarge"
                style={
                  Object {
                    "opacity": undefined,
                  }
                }
              />
              <div
                className="react-reveal mt-4 pr-5"
                style={
                  Object {
                    "opacity": undefined,
                  }
                }
              >
                <form>
                  <div
                    className="text-left nxt_body-xsmall"
                  >
                    <div
                      className="form-group"
                    >
                      <label
                        htmlFor="Name"
                      >
                        Name
                      </label>
                      <input
                        className="form-control"
                        id="InputName"
                        type="text"
                      />
                      <label
                        className="mt-3"
                        htmlFor="Email"
                      >
                        Email
                      </label>
                      <input
                        className="form-control"
                        id="InputEmail"
                        type="email"
                      />
                      <label
                        className="mt-3"
                        htmlFor="Phone"
                      >
                        Phone
                      </label>
                      <input
                        className="form-control"
                        id="InputPhone"
                        type="phone"
                      />
                      <label
                        className="mt-3"
                        htmlFor="Message"
                      >
                        Message
                      </label>
                      <textarea
                        className="form-control"
                        id="InputMessage"
                        rows="3"
                        type="text"
                      />
                    </div>
                    <div
                      className="mt-5"
                    >
                      <a
                        href="/#"
                        onClick={[Function]}
                      >
                        <button
                          className="btn btn-primary rounded-pill nxt_body-small px-4 py-2"
                        >
                          Submit
                        </button>
                      </a>
                    </div>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
